### DISCUSSION TITLE
Users cannot be created during installation because the hosted database does not allow foreign key constraints

### DISCUSSION THREAD

**Discussion by BloodBladeW** [FreshRSS/FreshRSS#5683] [Q&A] [answered]
**Describe the bug**
I used an externally hosted MySQL for my installation. I found that I am unable to create users during the installation. 

**To Reproduce**
./cli/create-user.php --user username --password 'password'
[error] --- FreshRSS_UserDAO::createUser error: ["HY000",1105, "VT10001: foreign key constraints are not allowed"]

**Expected behavior**
I would like to have a way to bypass this restriction to create a user.

**Environment information (please complete the following information):**
- Device: AWS Lightsail
- OS: Debian 11
- FreshRSS version: 1.21.0
- Database version: VItess MySQL
- PHP version: 7.4.33
- Installation type: Docker

**Additional context**
I found that the database providers (PlanetScale and Vitess) claim that [using foreign key constraints conflicts with clustered MySQL](https://vitess.io/blog/2021-06-15-online-ddl-why-no-fk/). Is there any way I can get around this restriction?

**Comment by Alkarex** [comment]
Would you be able to try with our built-in SQLite instead of MySQL?

**Comment by BloodBladeW** [comment]
> Would you be able to try with our built-in SQLite instead of MySQL?

I'm relatively new to SQLite, what are its performance requirements? My server performance is weak. It has slow disk I/O. And CPU and memory resources are low.

**Comment by Alkarex** [comment, marked as answer]
Try with our built-in SQLite instead of MySQL.
SQLite should be just fine, also on small servers

**Comment by BloodBladeW** [comment]
Ok, I changed the database I was using and it looks great now.
